projects/optic/src/commands/capture/init.ts

```diff
@@ -0,0 +1,75 @@
+import yaml from 'yaml';
+import { updateOpticConfig } from '../../utils/write-optic-config';
+import { logger } from '../../logger';
+
+export async function initCaptureConfig(
+  oasFile: string,
+  skipConfigUpdate: boolean,
+  opticConfigPath: string
+) {
+  const captureExample = captureConfigExample(oasFile);
+  const parsedExample = yaml.parseDocument(captureExample);
+
+  if (skipConfigUpdate) {
+    // console.log() to skip any future formatting changes we might make to logger
+    console.log(parsedExample.toString());
+    return;
+  }
+  logger.info(`Writing capture config to ${opticConfigPath}`);
+  try {
+    await updateOpticConfig(parsedExample, opticConfigPath);
+  } catch (err) {
+    throw err;
+  }
+}
+
+// returns a complete Capture block example
+export function captureConfigExample(oasFile: string) {
+  return `
+    capture:
+      ${oasFile}:
+        config:
+          # the number of parallel requests to make when using 'requests.send'
+          # optional, default: 4
+          request_concurrency: 4
+        server:
+          # a command to run your server
+          # optional, if ommitted Optic assumes the server is running or started elsewhere
+          command: your-server-command
+          # the url where your server can be reached once running
+          # required, can be overridden with '--server-override'
+          url: http://localhost:8080
+          # a readiness endpoint for Optic to validate before sending requests
+          # optional, default: /
```

Comment:
I don't think we're defaulting this value to `/` - if you don't pass us anything we don't hit the endpoint

Reply:
you're right — i'll update that comment